site/specification/ORCv2.md

```diff
@@ -1257,6 +1277,19 @@ Because dictionaries are accessed randomly, there is not a position to
 record for the dictionary and the entire dictionary must be read even
 if only part of a stripe is being read.
 
+Note that for columns with multiple streams, the order of these
+streams is **fixed**, which is different from data stream we mentioned above.
```

Comment:
```suggestion
Note that for columns with multiple streams, the order of stream
positions in the RowIndex is **fixed**, which may be different to
the actual data stream placement.
```

Reply:
We should also mention that the order is the same as what `Column Encodings` section has described for each column type.

Reply:
It looks resonable and will add more description/remove the chart.